src/Components/PlayBar/Components/PlayBarMain/Components/PlayListBox/PlayListBox.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import theme, { flexcenter } from "../../../../../../styles/Theme";
+import styled from "styled-components";
+import AlbumInfo from "../../../AlbumInfo/AlbumInfo";
```

Comment:
```suggestion
import React from "react";
import styled from "styled-components";
import AlbumInfo from "../../../AlbumInfo/AlbumInfo";
import theme, { flexcenter } from "../../../../../../styles/Theme";
```
기본적으로 컨벤션 잘 맞춰주셔야 읽기 좋고 협업하기 좋은 코드가 됩니다.

Reply:
수정하였습니다 ~